### DISCUSSION TITLE
mod_auth_openidc on oracle Linux 7 httpd

### DISCUSSION THREAD

**Discussion by chavalimk** [OpenIDC/mod_auth_openidc#596] [Q&A]
Has anyone managed to install this module on the httpd in oracle linux 7?

Can someone give me the commands to download the dependencies and install them please?

**Comment by philippe8888** [comment]
https://github.com/zmartzone/mod_auth_openidc/releases/download/v2.4.0/cjose-0.6.1.5-2.el7.x86_64.rpm
rpm -Uvh cjose-0.6.1.5-2.el7.x86_64.rpm
https://rpmfind.net/linux/epel/7/x86_64/Packages/h/hiredis-0.12.1-2.el7.x86_64.rpm
rpm -Uvh hiredis-0.12.1-2.el7.x86_64.rpm
yum install pcre2
https://github.com/zmartzone/mod_auth_openidc/releases/download/v2.4.11/mod_auth_openidc-2.4.11-1.el7.x86_64.rpm
rpm -Uvh mod_auth_openidc-2.4.11-1.el7.x86_64.rpm
